docs/advanced-features/private-endpoint-access.md

```diff
@@ -0,0 +1,130 @@
+<!-- markdownlint-disable -->
+# Private Endpoint Access
+
+## The Data Perimeter concept
+
+Establishing a **data perimeter** pattern that only allows access to trusted
+principals from trusted networks is a best practice to guarantee your
+organization's data security. A data perimeter helps protect your data
+from unintended access and potential configuration errors via
+built-in barriers.
+
+You can restrict Amazon S3 bucket access to a particular VPC and VPN traffic
+via a data perimeter pattern, which prevents leaked S3 credentials from 
+bypassing your organization's VPN.
+
+> Quilt already has private IPs for all Quilt services (Lambda
+functions, API Gateway, Quilt catalog API).
+
+To implement a data perimeter, you will need to take the following steps.
+
+## 1. Configure and deploy a Service Control Policy and Amazon S3 bucket policy
```

Comment:
Probably this item should go last because it requires S3 gateway endpoint.

Reply:
Good catch. Done.